crates/ruff/src/checkers/ast/deferred.rs

```diff
@@ -0,0 +1,20 @@
+use rustpython_parser::ast::{Expr, Stmt};
+
+use crate::ast::types::RefEquality;
+use crate::checkers::ast::AnnotationContext;
+use crate::docstrings::definition::Definition;
+use crate::visibility::{Visibility, VisibleScope};
+use crate::Range;
+
+type Context<'a> = (Vec<usize>, Vec<RefEquality<'a, Stmt>>);
+
+#[derive(Default)]
+pub struct Deferred<'a> {
```

Comment:
Nit: Can you add some documentation explaining what a `Deferred` is? I'm unable to guess its usage from just its name.